.github/workflows/spdx_license_bot.yaml

```diff
@@ -0,0 +1,64 @@
+name: spdx_license_bot
+
+on:
+  # This should ideally trigger whenever there is a commit to the [SPDX License repository](https://github.com/spdx/license-list-data).
+  # However, this is not yet possible see: https://github.com/orgs/community/discussions/26323
+  schedule:
+    # At 08:04 on every day-of-week from Monday through Friday.
+    - cron: "4 8 * * 1-5"
+  workflow_dispatch:
+
+env:
+  # The code Mason returns when there are changes detected after generation, see:
+  # https://github.com/felangel/mason/tree/master/packages/mason_cli#detecting-changes-after-generation
+  MASON_DETECTED_CHANGES_CODE: 70
+
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: tool/spdx_license
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 📚 Git Checkout
+        uses: actions/checkout@v4
+
+      - name: 🎯 Setup Dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: 3.1.0
+
+      - name: 📦 Install Dependencies
+        run: dart pub get
+
+      - name: 💻 Install Mason
+        run: |
+          dart pub global activate mason_cli
+          mason get
+
+      - name: 🧱 Mason Make
+        id: make
+        run: echo "MAKE_CODE=${mason make spdx_license --licenses "[]" -o test --on-conflict overwrite --set-exit-if-changed}" >> $GITHUB_ENV
+
+      - name: 🔑 Config Git User
+        if: $MAKE_CODE == $MASON_DETECTED_CHANGES_CODE
+        run: |
+          git config user.name VGV Bot
+          git config user.email vgvbot@users.noreply.github.com
+
+      - name: 📝 Create Pull Request
+        if: $MAKE_CODE == $MASON_DETECTED_CHANGES_CODE
+        uses: peter-evans/create-pull-request@v5.0.2
+        with:
+          base: main
+          branch: chore/update-spdx-license
+          commit-message: "chore: update SPDX licenses"
+          title: "chore: update SPDX licenses"
+          body: Please squash and merge me!
+          labels: bot
+          author: VGV Bot <vgvbot@users.noreply.github.com>
+          assignees: vgvbot
+          reviewers: alestiago
```

Comment:
I would keep this empty and just have the CODEOWNER file trigger

Reply:
Awesome, I will also edit https://github.com/VeryGoodOpenSource/very_good_cli/blob/2a7ad1e0d3e2a471036b0ecf0bffd91cfcd99aa3/.github/workflows/bump_templates.yaml#L39.